app/controllers/admin/site_settings_controller.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class Admin::SiteSettingsController < Admin::AdminController
+  MAX_NOTIFICATION_COUNT = 10000
```

Comment:
```suggestion
  MAX_NOTIFICATION_COUNT ||= 10000
```
This is a handy trick in case the case is re-evaluated for any reason.